mu4/scenes/notation/view/notationpaintview.cpp

```diff
@@ -391,3 +417,24 @@ qreal NotationPaintView::scale() const
 {
     return QQuickPaintedItem::scale();
 }
+
+void NotationPaintView::onPlayingChanged()
+{
+    bool isPlaying = playbackController()->isPlaying();
+    m_playbackCursor->setVisible(isPlaying);
+
+    if (isPlaying) {
+        m_playbackUpdateTimer.start();
```

Comment:
why are you using a timer instead of a notification? isPlayingChanged

Reply:
This may change depending on the requirements.
If we decide to draw the cursor discretely by events that are being played (as in MU3), then we can use `isPlayingChanged`, but if we decide to draw the cursor so that it moves smoothly, then we need a timer and additional calculation of the cursor position because `isPlayingChanged` will be triggered too rarely for smooth movement. 
This issue will be resolved through one pull request.